background.ts

```diff
@@ -973,7 +967,7 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
   ): Promise<[string, string]> {
     const [needToUpdate, errors] = this.settingsValidator.validateTransientSettings(TransientSettings.value, newTransientSettings);
 
-    return Promise.resolve((() => {
+    return Promise.resolve((():[string, string] => {
```

Comment:
It's in a separate PR, but my IDE has been complaining about a type-mismatch the whole time. 

I know that the company behind this IDE is lagging behind the tslint/eslint we use, but it would be
nice to at least not have this warning.